client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -56,9 +58,24 @@ const MetadataEditor = createClass({
 		});
 	},
 
+
+	ThumbnailCapture : async function() {
+		const bR = parent.document.getElementById('BrewRenderer');
+		const brewRenderer = bR.contentDocument || bR.contentWindow.document;
```

Comment:
Curious why this is an `OR` between `contentDocument` and `contentWindow.document`?  From what I can tell, it is the same thing?

Reply:
My memory of the reference code is that it should be the same but for some browser situations, it may not be.